pkg/okteto/kubetoken_client.go

```diff
@@ -80,7 +80,7 @@ func (c *kubeTokenClient) GetKubeToken(baseURL, namespace string) (types.KubeTok
 		return types.KubeTokenResponse{}, fmt.Errorf("failed to unmarshal kubetoken response: %w", err)
 	}
 
-	return kubeTokenResponse, nil
+	return kubeTokenResponse, resp.Body.Close()
```

Comment:
Could we  move the `resp.Body.Close` to the line before and log it into the log file?

We are checking that `err != nil` from the calling function. If we get the response correctly but the body is not able to close we will be returning an error to the user that we could have ignore.

```suggestion
        defer func() {
          if err := resp.Body.Close(); err != nil {
            oktetoLog.Info("could not close the body: %s", err)
          }
        }
	return kubeTokenResponse, nil
```